app/javascript/dashboard/components/ChatList.vue

```diff
@@ -544,6 +552,11 @@ export default {
       this.$store.dispatch('customViews/update', payloadData);
       this.closeAdvanceFiltersModal();
     },
+    setSavedSortByFilter() {
+      const { status, order_by: orderBy } = this.savedSortByFilter || {};
```

Comment:
Instead of handling this here, let's ensure that `savedSortBytFilter` always returns an object

```suggestion
      const { status, order_by: orderBy } = this.savedSortByFilter;
```